docs/current/guides/snippets/use-services/expose-service-containers-to-host/main.py

```diff
@@ -4,6 +4,8 @@
 
 import dagger
 
+import httpx
```

Comment:
These imports are not sorted, so you'll get a linting failure. It's builtins, then third party than own, so:

```
import sys

import anyio
import dagger
import httpx
```

Depending on how the sdk is installed, dagger could go on its own line in the bottom. What matters is ci, just run `./hack/make sdk:python:lint`.

To re-format locally, having `hatch` installed, it's just `hatch run lint`.

Reply:
I can't get `hatch run lint` to work `line 1: lint: command not found`

And linting is failing for both:
```python
import sys

import anyio
import dagger
import httpx
```

and


```python
import sys

import anyio
import httpx

import dagger
```

Any idea?

Reply:
Hatch needs to be run from `sdk/python`.

Reply:
Pushed